test/controllers/roles_controller_test.rb

```diff
@@ -123,5 +123,15 @@ class RolesControllerTest < ActionController::TestCase
       assert_not_nil cloned_role
       assert_equal @role.permissions, cloned_role.permissions
     end
+
+    test "cloned roles should have builtin value 0 instead of `false`" do
```

Comment:
The comment about `false` isn't helpful after this patch is merged, it should simply state that it should have the value 0.